src/ucp/rndv/rndv.c

```diff
@@ -1091,6 +1122,12 @@ ucp_rndv_recv_start_get_pipeline(ucp_worker_h worker, ucp_request_t *rndv_req,
     ucp_rndv_req_init_zcopy_lane_map(rndv_req, rndv_req->send.mem_type,
                                      UCP_REQUEST_SEND_PROTO_RNDV_GET);
 
+    lane = ucp_rndv_zcopy_get_lane(rndv_req, &uct_rkey,
+                                    UCP_REQUEST_SEND_PROTO_RNDV_GET);
+    if (lane == UCP_NULL_LANE) {
+        goto err;
+    }
```

Comment:
when can it be NULL?

Reply:
like in normal GET Zcopy, it can be NULL if not able to select GET protocol due to RKEY
I had a reproducer for this. So, it was fixed in the same as in GET

Reply:
what is the reproducer?